src/dbPool/schema/UserSchema.ts

```diff
@@ -302,3 +302,33 @@ class UserChangePasswordVerificationCodeSchemaFactory {
 	schemaInstance = new Schema(this.schema)
 }
 export const UserChangePasswordVerificationCodeSchema = new UserChangePasswordVerificationCodeSchemaFactory()
+
+/**
+ * 用户身份验证器
+ */
+class UserAuthenticatorSchemaFactory {
+	/** MongoDB Schema */
+	schema = {
+		/** 用户的 UUID，关联用户安全集合的 UUID - 非空 */
+		UUID: { type: String, required: true },
```

Comment:
接上条，没有考虑用户已经绑定过身份验证器的情况。
增加索引比较好
```suggestion
		UUID: { type: String, required: true, unique: true },
```